src/tools/generate/generate_chapters.js

```diff
@@ -51,6 +51,9 @@ const parse_file = async (markdown,chapter) => {
   const html = converter.makeHtml(markdown);
   let body = html;
 
+  // Make headers links
+  const header_regex = /<h([2-6]) id="(.*?)">(.*?)<\/h[2-6]>/gi;
+  body = body.replace(header_regex, "<h$1 id=\"$2\"><a href=\"#$2\" class=\"anchor-link\">$3</a></h$1>");
```

Comment:
Implement this in an external function for consistency with other transformations

Reply:
Fixed